extensions/spark/kyuubi-spark-authz-bundle/pom.xml

```diff
@@ -0,0 +1,317 @@
+                            <include>org.apache.ranger:ranger-plugins-common</include>
+                            <include>org.apache.ranger:ranger-plugins-audit</include>
+                            <include>org.codehaus.jackson:jackson-jaxrs</include>
+                            <include>com.sun.jersey:jersey-client</include>
+                            <include>com.sun.jersey:jersey-core</include>
+                            <include>com.kstruct:gethostname4j</include>
+                            <include>net.java.dev.jna:jna</include>
+                            <include>net.java.dev.jna:jna-platform</include>
+                        </includes>
+                    </artifactSet>
+                    <filters>
+                        <filter>
+                            <artifact>*:*</artifact>
+                            <excludes>
+                                <exclude>**/*.proto</exclude>
+                                <exclude>META-INF/*.SF</exclude>
+                                <exclude>META-INF/*.DSA</exclude>
+                                <exclude>META-INF/*.RSA</exclude>
+                                <exclude>META-INF/DEPENDENCIES</exclude>
+                                <exclude>META-INF/LICENSE.txt</exclude>
+                                <exclude>META-INF/NOTICE.txt</exclude>
+                                <exclude>META-INF/maven/**</exclude>
+                                <exclude>LICENSE.txt</exclude>
+                                <exclude>NOTICE.txt</exclude>
+                                <exclude>mozilla/**</exclude>
+                                <exclude>**/module-info.class</exclude>
+                            </excludes>
+                        </filter>
+                    </filters>
+                    <relocations>
+                        <relocation>
+                            <pattern>org.codehaus.jackson.jaxrs</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.org.codehaus.jackson.jaxrs</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>com.sun.jersey</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.com.sun.jersey</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>com.sun.ws.rs.ext</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.com.sun.ws.rs.ext</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>com.kstruct.gethostname4j</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.com.kstruct.gethostname4j</shadedPattern>
+                        </relocation>
+                        <relocation>
+                            <pattern>org.apache.hadoop.security</pattern>
```

Comment:
does it mean the shaded jar contains hadoop classes? it's crazy

Reply:
No, this just contains the following three classes, which are in ranger-plugins-common;

<img width="584" alt="image" src="https://github.com/apache/kyuubi/assets/51110188/adb0dca7-d3b9-4219-a2c2-2879a01d3f5c">


<img width="1232" alt="image" src="https://github.com/apache/kyuubi/assets/51110188/dd7a5320-dfe0-465f-8194-dc9c1653b14c">

Reply:
oh, it comes from ranger, that's fine